dist/smartbanner.css

```diff
@@ -1,7 +1,3 @@
-/*!
- * smartbanner.js v1.5.0 <https://github.com/ain/smartbanner.js>
- * Copyright © 2017 Ain Tohvri, contributors. Licensed under GPL-3.0.
- */
```

Comment:
Had to remove '&& npm run bannerise' during the development, because I am working on windows and was getting the error during npm build during execution of this script, which is why this was automatically created without the banner during build process.